olive/systems/docker/docker_system.py

```diff
@@ -169,7 +170,12 @@ def _run_container(self, tempdir, model: OliveModel, metrics: List[Metric], cont
                     command=f"python {clean_up_mount_path} --dev_mount_path {dev_mount_path}",
                     volumes=[dev_mount_str, clean_up_mount_str],
                 )
-                logger.debug("Dev mount cleaned up successfully")
+
+        if docker.version_info[0] >= 3:
```

Comment:
why need this check?

Reply:
For docker which version less that 3, the exit_code is integer not dict. 
But I double checked the docker version in Olive, which should be the latest 6.x.x where the exit_code is dictionary. 

Removed this check.